python/cudf/cudf/core/series.py

```diff
@@ -1210,6 +1210,14 @@ def drop_duplicates(self, keep="first", inplace=False):
 
         return self._mimic_inplace(result, inplace=inplace)
 
+    def rank(self, axis=0, method="average", numeric_only=None, na_option="keep", ascending=True, pct=False):
+        """
+        Compute numerical data ranks (1 through n) along axis.
+
+        By default, equal values are assigned a rank that is the average of the ranks of those values.
+        """
+        return super().rank(axis, method, numeric_only, na_option, ascending, pct)
```

Comment:
Since we're not specializing `rank` in any way for `Series`, we don't need this call to `super()`. We can get rid of Series level `rank` entirely - it will just get inherited from `Frame`.